recipes/gwcelery/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "gwcelery" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bf3baac0e3a36d0aa1250fd0ab130990ff9245e2b8295012993ea67c4d5c19e8
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  entry_points:
+    - gwcelery = gwcelery.app:start
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools >=30.3.0
+  run:
+    - astropy
+    - celery >=4.2.0
+    - celery-eternal >=0.0.3
+    - comet
+    - flower
+    - gwdatafind
+    - gwpy
+    - healpy
+    - jinja2
+    - lalsuite >=6.52
+    - ligo-followup-advocate >=0.0.11
+    - ligo-gracedb >=2.1.1
+    - ligo-raven >=1.8
+    - ligo-segments
+    - ligo.skymap >=0.1.0
+    - lscsoft-glue
+    - lxml
+    - numpy
+    - pygcn >=0.1.16
+    - python >=3.6
+    - redis-py
+    - safe-netrc
+    - sentry-sdk
+    - service_identity
+    - sleek-lvalert
+    - voeventlib >=1.2
+
+test:
+  imports:
+    - gwcelery
+  requires:
+    - pytest >=3.0
+    - pytest-socket
+  commands:
+    - gwcelery --help
+    - python -m pytest --pyargs $PKG_NAME
+
+about:
+  home: https://git.ligo.org/emfollow/gwcelery
+  license: GPLv2+
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.md'
```

You can delete the licence file and get it from the PyPI tarball instead.

```suggestion
  license_file: doc/license.md
```

---

thanks @chrisburr, done.